### ISSUE TITLE
PHP memory exhausted

### ISSUE THREAD

**Issue by ncfernan** [AzuraCast/AzuraCast#3979]
**Using Docker installation method**
Yes

**AzuraCast version**
Rolling Release #e0270f9 (2021-04-06 11:32) • Docker • PHP 7.4

**Host Operating System**
Ubuntu 20.04 LTS

**Describe the bug**
I was using the stable version but the CPU usage was near 100%, so I decided to switch to latest version. After that, the "now playing" task does not run anymore. When I run that task manually (after 9 hours), I receive the following error message:

[2021-04-06T12:26:53.002845+00:00] AzuraCast.CRITICAL: Fatal error: Allowed memory size of 536870912 bytes exhausted (tried to allocate 634880 bytes) in /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php on line 112 [] {"station":{"id":1,"name":"NCF Radio"}}

The 512M limit was set in "azuracast.env" because the original 256M was triggering the same error message.

I just updated the installation and rebooted the machine.

**To Reproduce**
Just start Azuracast

**Expected behavior**
The "now playing" task running smoothly

**Relevant Logs**
Loading...[2021-04-06T12:26:51.593651+00:00] AzuraCast.NOTICE: Running sync task: Now Playing Data {"force":true} []
[2021-04-06T12:26:51.599286+00:00] AzuraCast.DEBUG: Successfully acquired the "{resource}" lock. {"resource":{"Symfony\\Component\\Lock\\Key":"lock_sync_nowplaying"}} []
[2021-04-06T12:26:51.600247+00:00] AzuraCast.DEBUG: Expiration defined for "{resource}" lock for "{ttl}" seconds. {"resource":{"Symfony\\Component\\Lock\\Key":"lock_sync_nowplaying"},"ttl":600.0} []
[2021-04-06T12:26:51.627312+00:00] AzuraCast.DEBUG: Starting sub-task: BuildQueueTask [] []
[2021-04-06T12:26:51.641975+00:00] AzuraCast.DEBUG: Successfully acquired the "{resource}" lock. {"resource":{"Symfony\\Component\\Lock\\Key":"lock_autodj_queue_1"}} []
[2021-04-06T12:26:51.642273+00:00] AzuraCast.DEBUG: Expiration defined for "{resource}" lock for "{ttl}" seconds. {"resource":{"Symfony\\Component\\Lock\\Key":"lock_autodj_queue_1"},"ttl":60.0} []
[2021-04-06T12:26:51.798635+00:00] AzuraCast.DEBUG: Adding to station queue. {"now":"2021-04-06 09:30:29"} {"station":{"id":1,"name":"NCF Radio"}}
[2021-04-06T12:26:51.838528+00:00] AzuraCast.INFO: AzuraCast AutoDJ is calculating the next song to play... [] {"station":{"id":1,"name":"NCF Radio"}}
[2021-04-06T12:26:53.002845+00:00] AzuraCast.CRITICAL: Fatal error: Allowed memory size of 536870912 bytes exhausted (tried to allocate 634880 bytes) in /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php on line 112 [] {"station":{"id":1,"name":"NCF Radio"}}

**Screenshots**

**Device(s):**
- Device: Desktop
- OS: Win10
- Browser: chrome
- Version: 89.0.4389.114 (Official Build) (64-bit)

**Additional context**

**Comment by ErnestPH** [comment]
When doing` free -m` in terminal. How much is the free and total Mem? Did you also added Swap?

**Comment by ncfernan** [comment]
```
ncf-radio:~$ free -m
              total        used        free      shared  buff/cache   available
Mem:           1987         703         480          39         803        1077
Swap:           511         376         135
```

I installed using an image from Linode with Azuracast pre-installed (docker).

It was running it yesterday using the stable version. The CPU consumption was too high, toping 98% with two stations, so I switched to rolling release and it droped to 45%, but after half an hour the "now playing" task crashed with 256M as the limit for PHP operation.

I changed the "azuracast.env" file to raise that PHP limit to 512M, to no avail. I guess there is a loop that may have sneaked in the code.

Now I have a high CPU usage radio with some broken tasks...  bad move...

![image](https://user-images.githubusercontent.com/10008844/113783014-777b4800-9709-11eb-9b92-c75906c6d940.png)

![image](https://user-images.githubusercontent.com/10008844/113783240-d5a82b00-9709-11eb-8ea8-73661254f7d3.png)
